src/main/scala/is/hail/table/Table.scala

```diff
@@ -409,7 +409,7 @@ class Table(val hc: HailContext, val tir: TableIR) {
         keyBy(signature.fieldNames).keyedRDD().groupByKey().fullOuterJoin(
           other.keyBy(other.signature.fieldNames).keyedRDD().groupByKey()
         ).forall { case (k, (v1, v2)) =>
-          (v1, v2) match {
+          ((v1, v2): @unchecked) match {
```

Comment:
here, I prefer `case (None, None) => assert(false)` because if a refactor changes something this is a more glaring piece of code to change rather than this catch-all unchecked annotation.